### ISSUE TITLE
Error with entry file paths beginning with ./

### ISSUE THREAD

**Issue by maoberlehner** [rollup/rollup-plugin-node-resolve#43]
Does not work:
```bash
rollup --config --output app/js/index.js ./js/index.js
```

Works:
```bash
rollup --config --output app/js/index.js js/index.js
```

rollup.config.js
```node
import commonjs from 'rollup-plugin-commonjs';
import nodeResolve from 'rollup-plugin-node-resolve';

export default {
  plugins: [
    nodeResolve(),
    commonjs()
  ]
};
```

Error message:
```
Path must be a string. Received undefined
TypeError: Path must be a string. Received undefined
    at assertPath (path.js:7:11)
    at Object.resolve (path.js:1148:7)
    at resolveId$1 (/Users/moberlehner/Sites/vagrant/devbox/modularis/node_modules/rollup-plugin-node-resolve/dist/rollup-plugin-node-resolve.cjs.js:42:15)
    at /Users/moberlehner/Sites/vagrant/devbox/modularis/node_modules/rollup/src/utils/first.js:8:31
    at process._tickCallback (internal/process/next_tick.js:103:7)
    at Function.Module.runMain (module.js:577:11)
    at startup (node.js:160:18)
    at node.js:456:3
```

Is this an error or should I use paths **not** beginning with `./` in general?

**Comment by lammas** [comment]
Should we move to not using `./` in import paths or will this be fixed?

**Comment by TrySound** [comment]
Fixed in 1.7.2
